tests/data/yaml_file_with_not_yaml_extension

```diff
@@ -0,0 +1,12 @@
+# http://editor2.swagger.io/spec-files/minimal.yaml
```

Comment:
If we have to check for the file format we should consider the case in which the file is not JSON and not YAML according the different criteria used for discrimination (file extension and content type)